superset/migrations/versions/2023-03-27_12-30_7e67aecbf3f1_chart_ds_constraint.py

```diff
@@ -63,9 +63,15 @@ def upgrade():
     session = db.Session(bind=bind)
 
     with op.batch_alter_table("slices") as batch_op:
-        for slc in session.query(Slice).filter(Slice.datasource_type == "query").all():
-            upgrade_slc(slc)
-            session.add(slc)
+        for slc in session.query(Slice).filter(Slice.datasource_type != "table").all():
+            if slc.datasource_type == "query":
+                upgrade_slc(slc)
+                session.add(slc)
+
+            else:
+                print(
```

Comment:
Can we use a logger here? Print is a little nasty for aggregators like Datadog :P